GliaWidgets/SecureConversations/SecureConversations.PendingInteraction.swift

```diff
@@ -9,21 +9,34 @@ extension SecureConversations {
         private(set) var pendingStatusCancellationToken: String?
```

Comment:
Since you added error throwing, does it mean that properties storing token can be non-optional?

Reply:
@EgorovEI  Seems like it is not possible, because `weak self` has to be in capture list of the closure, and because of that subscription tokens must be initialised with some values, either `nil` or empty string.
```
let pendingStatusCancellationToken = environment.observePendingSecureConversationsStatus { [weak self] result in
     // in order to access weak `self`, all of it's value must be initialised.
     guard let self else { return }
     // ....
     pendingStatus = (try? result.get()) ?? false
    }
```

Reply:
Oh, I see. Then let's leave them optional